investigator/overlays/test/kustomization.yaml

```diff
@@ -5,8 +5,6 @@ commonLabels:
 resources:
   - ../../base
   - thoth-notification.yaml
-patchesStrategicMerge:
```

Comment:
why is it removed?

Reply:
is it required? if the image is in core? what patch we need to apply here?

Reply:
ja